### PR TITLE
feat: remove jquery

### PR DESCRIPTION
**Pull request by seonim-ryu**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

### Description

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/editor/src/js/utils/dom.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  }
+}
+
+export function toggleClass(element, className) {
+  if (element) {
+    element.classList.toggle(className);
+  }
+
+  return null;
+}
```
Comment (seonim-ryu) [inline apps/editor/src/js/utils/dom.js]: `./src/js/domUtils.js`로 이동
> https://github.com/nhn/tui.editor/pull/753/commits/01613fac63aaf59749cbf70013970b70b3ae4004

File: apps/editor/src/js/codeBlockPreview.js [hunk 2]
```diff
@@ -2,21 +2,22 @@
  * @fileoverview Implements CodeBlockPreview
  * @author NHN FE Development Lab <dl_javascript@nhn.com>
  */
+import $ from 'jquery';
```
Comment (eliashdezr) [inline apps/editor/src/js/codeBlockPreview.js]: Should be this removed if we want to deprecate `jquery`?

Comment (seonim-ryu) [reply apps/editor/src/js/codeBlockPreview.js]: @elamperti I'm gradually removing JQuery, and this is the code to add temporarily as needed. And this PR is for tracking work, so I don't need to review it.  😅

Comment (eliashdezr) [reply apps/editor/src/js/codeBlockPreview.js]: @seonim-ryu my bad :)

**Pull request closed without merging**